uktrade/data-hub-frontend
# Add ErrorSummary component to fix accessibility issues

## Description of change
After an audit from the DAC (Digital Accessibility Centre) we need to address issues around the error message skip links. When a user submits a form and receives errors the focus should then go to the error summary of which contains links to each error, each link should anchor to the appropriate input field.

## Solution
I realised that the mark-up in the component we use from the "govuk-react" module is wrong and is missing href attributes so I have ported this component over to our repo and modified it so we can be address the issues and become AA compliant.

## Test instructions
- Go to a form such as "Add to pipeline/" or "Add referral"
- Submit with errors

The error links should anchor to the appropriate input field.

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
